Loader/SymfonyFixturesLoader.php

```diff
@@ -67,10 +66,6 @@ public function addFixture(FixtureInterface $fixture): void
             $this->addGroupsFixtureMapping($class, $fixture::getGroups());
         }
 
-        if ($fixture instanceof ContainerAwareInterface) {
-            $fixture->setContainer($this->container);
-        }
```

Comment:
In this file the `$container` I guess is not needed anymore

Reply:
Deprecated in https://github.com/doctrine/DoctrineMongoDBBundle/pull/801